src/react-native/13-making-http-requests/making-http-requests.md

```diff
@@ -106,17 +114,151 @@ If you’ve implemented the solution correctly, the tests will pass when you run
+
+### Exercise 3
+
+TODO
+
+### Solution 3
+
+If you’ve implemented the solution correctly, the tests will pass when you run `npm run test`!
+
+<details>
+<summary>Click to see the solution</summary>
+
+✏️ Update **src/services/pmo/api/api.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-problem/src/services/pmo/api/api.ts ../../../exercises/react-native/13-http-requests/03-solution/src/services/pmo/api/api.ts only
+
+✏️ Update **src/services/pmo/restaurant/hooks.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-problem/src/services/pmo/restaurant/hooks.ts ../../../exercises/react-native/13-http-requests/03-solution/src/services/pmo/restaurant/hooks.ts only
+
+</details>
+
+## Objective 4: `fetch` restaurants in a custom hook
+
+### Setup 4
+
+✏️ Update **src/services/pmo/restaurant/hooks.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/services/pmo/restaurant/hooks.ts ../../../exercises/react-native/13-http-requests/04-problem/src/services/pmo/restaurant/hooks.ts only
+
+✏️ Create **src/services/pmo/restaurant/interfaces.ts** and update it to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/services/pmo/restaurant/interfaces.ts ../../../exercises/react-native/13-http-requests/04-problem/src/services/pmo/restaurant/interfaces.ts only
+
+✏️ Update **src/screens/RestaurantList/RestaurantList.tsx** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/screens/RestaurantList/RestaurantList.tsx ../../../exercises/react-native/13-http-requests/04-problem/src/screens/RestaurantList/RestaurantList.tsx only
+
+✏️ Update **src/screens/RestaurantDetails/RestaurantDetails.tsx** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/screens/RestaurantDetails/RestaurantDetails.tsx ../../../exercises/react-native/13-http-requests/04-problem/src/screens/RestaurantDetails/RestaurantDetails.tsx only
+
+### Verify 4
+
+✏️ Update **src/services/pmo/restaurant/hooks.test.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/services/pmo/restaurant/hooks.test.ts ../../../exercises/react-native/13-http-requests/04-problem/src/services/pmo/restaurant/hooks.test.ts only
+
+✏️ Update **src/screens/RestaurantList/RestaurantList.test.tsx** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/screens/RestaurantList/RestaurantList.test.tsx ../../../exercises/react-native/13-http-requests/04-problem/src/screens/RestaurantList/RestaurantList.test.tsx only
+
+✏️ Update **src/screens/RestaurantDetails/RestaurantDetails.test.tsx** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/03-solution/src/screens/RestaurantDetails/RestaurantDetails.test.tsx ../../../exercises/react-native/13-http-requests/04-problem/src/screens/RestaurantDetails/RestaurantDetails.test.tsx only
+
+### Exercise 4
+
+TODO
```

Comment:
last one 😅😅

Reply:
Yep, creating exercises is in the content portion